### DISCUSSION TITLE
Dbeaver can read the results of unbounded streams ?

### DISCUSSION THREAD

**Discussion by waywtdcc** [dbeaver/dbeaver#19443] [General]
Hello everyone, I would like to ask whether dbeaver can read the results of unbounded streams without stopping and keep querying?

**Comment by LonwoLonwo** [comment]
Hello @waywtdcc 

Please provide more info and examples about your question. What database do you use? What exactly are unbounded streams?

**Reply by waywtdcc** [reply]
I am querying a dynamic table and the result is unbounded, for example using kyuubi flink. I hope that dbeaver can continuously refresh and display the queried data to the page without the end of the query.

**Reply by Destrolaric** [reply]
@waywtdcc Unfortunately, no, DBeaver isn't able to process stream data.
